src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/utils/TestTrustManager.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.coh.functional.bdd.utils;
+
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.conn.ssl.TrustStrategy;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.springframework.http.client.ClientHttpRequestFactory;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+
+import javax.net.ssl.SSLContext;
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.X509Certificate;
+
+public class TestTrustManager {
+
+    private static TestTrustManager INSTANCE;
+
+    private static final TrustStrategy acceptingTrustStrategy = (X509Certificate[] chain, String authType) -> true;
+
+    private TestTrustManager() {
+    }
+
+    synchronized public static TestTrustManager getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new TestTrustManager();
+        }
+
+        return INSTANCE;
+    }
+
+    public TrustStrategy getTrustStrategy() {
+        return acceptingTrustStrategy;
+    }
+
+    synchronized public ClientHttpRequestFactory getTestRequestFactory() throws KeyStoreException, NoSuchAlgorithmException, KeyManagementException {
```

Comment:
Spring rest template does accept a TrustStrategy on it's own. It requires a bit more boiler plate code, which is the ClientHttpRequestFactory that's being constructed here with the accept anything strategy.